vignettes/sass.Rmd

```diff
@@ -344,16 +344,32 @@ sass(
 
 ### Caching {#caching}
 
-Sometimes calling `sass()` can be computationally expensive, in which case, it can be useful to leverage its caching capability. Caching is automatically enabled in non-interactive sessions, but you can set the `sass.cache` option to `TRUE` to enable it universally, and/or use `sass_cache_options(cache = TRUE)` to enable for specific calls to `sass()`. See `sass_cache_options()` for more details about how caching works.
+Sometimes calling `sass()` can be computationally expensive, in which case, it can be useful to leverage its caching capability. Caching is enabled by default, unless Shiny's Developer Mode (`shiny::devmode()`) is enabled. To explicitly enable (disable), set `options(sass.cache = )` to `TRUE` (or `FALSE`):
+
+```r
+withr::with_options(
+  list(sass.cache = TRUE),
+  sass(sass_file("my-style.scss"))
+)
+```
+
+You can also configure the location, size, and age of file caching via `sass_file_cache()`, which can be passed directly to a `sass()` call:
 
 ```r
-options(sass.cache = TRUE)
 sass(
   sass_file("my-style.scss"),
-  cache_options = sass_cache_options(cache = TRUE)
+  cache = sass_file_cache(getwd(), max_size = 100 * 1024^2)
 )
 ```
 
+Or used with `sass_cache_set_dir()` to configure the file cache globally:
+
+```r
+sass_cache_set_dir(getwd(), sass_file_cache(getwd(), max_size = 100 * 1024^2))
```

Comment:
@wch do you remember why we didn't do `sass_cache_set(cache)` and get the directory from the `FileCache` object?

Reply:
I don't remember. It's possible that we did it to keep the signature similar to `sass_cache_get_dir()`. The signatures are:

```R
sass_cache_get_dir <- function(dir, create = FALSE)

sass_cache_set_dir <- function(dir, cache)
```

So the first argument is `dir` in both cases. With `sass_cache_set_dir()`, I suppose it might have been a little weird to call `sass_cache_set_dir()` and not actually pass it a directory name. But it is also a little weird to pass it a directory name that is redundant. 🤷